examples/antarctic_slope_current_model.jl

```diff
@@ -0,0 +1,154 @@
+
+#using CairoMakie
+using Oceananigans
+using Oceananigans.Units: minute, minutes, hour
+using SeawaterPolynomials.TEOS10
+
+# This file sets up a model that resembles the Antarctic Slope Current (ASC) model in the
+# 2022 paper by Ian Eisenman
+
+arch = GPU()
+
+g_Earth = 9.80665
```

Comment:
It might be simpler to keep the defaults rather than change this. I made a comment though if we want to manually set this